walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/requirements/SelfAttestedClaimRequirementTest.kt

```diff
@@ -35,23 +34,27 @@ class SelfAttestedClaimRequirementTest {
     }
 
     @Test
-    fun selfAttestedRequirement_validateUnFulfilledRequirement_ThrowsException() {
+    fun selfAttestedRequirement_validateUnFulfilledRequirement_ReturnsFailure() {
```

Comment:
nit: validate_WithFulfilledRequirement_ReturnsFailure

Reply:
will add the changes in the next PR.